openpype/settings/defaults/project_settings/global.json

```
@@ -397,11 +397,15 @@
             ]
         },
         "CleanUp": {
+            "enabled": true,
             "paterns": [],
             "remove_temp_renders": false
         },
         "CleanUpFarm": {
             "enabled": false
+        },
+        "ExplicitCleanUp": {
+            "enabled": false
```

It should be enabled by default.
```suggestion
            "enabled": true
```